packages/paste-website/src/components/tokens-list/index.tsx

```diff
@@ -125,30 +132,35 @@ export const TokensList: React.FC<TokensListProps> = (props) => {
     });
   };
 
+  React.useEffect(() => {
+    if (typeof filterString === 'string') filterTokenList(filterString);
+    trackTokenFilterString(filterString);
+  }, [filterString]);
+
   const handleInput = (e: React.FormEvent<HTMLInputElement>): void => {
     const filter = e.currentTarget.value;
     setFilterString(filter);
-    filterTokenList();
-    trackTokenFilterString(filter);
+    if (filter === '') navigate(window.location.pathname, {replace: true});
+    else navigate(`?tokens-filter=${filter}`, {replace: true});
   };
 
   const uid = useUID();
 
   return (
     <>
-      <Box as="form" marginTop="space100" marginBottom="space100" maxWidth="size40">
+      <Box as="div" marginTop="space100" marginBottom="space100" maxWidth="size40">
```

Comment:
The form has default behavior that was making the page reload and the input value clear on hitting the enter key, and the AC says that the enter key shouldn't be necessary to a search (or perform any actions really, since they should happen live on each keystroke, which they now are). I was under the impression that it's ok to have one search filter Input field outside of a form field (a11y-wise), is that true?

Reply:
Technically, it's still better to have it contained in a form for accessibility reason. You can navigate by forms with AT. 

You can prevent the default behaviour of the form element though, by adding 

`onSubmit={ error => error.preventDefault() }`

to the form. So when you press enter, you submit the form, but the forms default behaviour of reloading the page is prevented

Reply:
That was the first approach before changing it to a div, so I can definitely change it back to prevent default 👍